src/tbb/misc_ex.cpp

```diff
@@ -147,7 +149,7 @@ static void initialize_hardware_concurrency_info () {
         const int curMaskSize = BasicMaskSize * numMasks;
         processMask = new basic_mask_t[numMasks];
         std::memset( processMask, 0, curMaskSize );
-#if __linux__
+#if __unix__
```

Comment:
For some reason GitHub does not allow multiline comment with untouched code. Swap the Unix and *BSD blocks here as well.

Reply:
If I swap this, then unix would be the default case. Is that ok?

Reply:
It is acceptable, currently, the main issue that *BSD system will use __unix__ branch that will not work

Reply:
I have applied the suggestion including a minor simplification.